plugins/org.jboss.tools.openshift.js/src/org/jboss/tools/openshift/js/server/behaviour/OpenShiftNodejsLaunchController.java

```diff
@@ -16,19 +16,27 @@
 import org.jboss.ide.eclipse.as.wtp.core.server.behavior.ISubsystemController;
 import org.jboss.tools.openshift.core.server.OpenShiftServerBehaviour;
 import org.jboss.tools.openshift.core.server.behavior.OpenShiftLaunchController;
+import org.jboss.tools.openshift.internal.core.Dialogs;
 import org.jboss.tools.openshift.internal.core.server.debug.DebugContext;
 import org.jboss.tools.openshift.internal.core.server.debug.IDebugListener;
 import org.jboss.tools.openshift.internal.core.server.debug.OpenShiftDebugMode;
 import org.jboss.tools.openshift.js.launcher.NodeDebugLauncher;
 
 public class OpenShiftNodejsLaunchController extends OpenShiftLaunchController implements ISubsystemController {
 
+	private static final String WARN_NEWER_THAN_NODEJS7_UNSUPPORTED = "org.jboss.tools.openshift.js.dontwarnNodeJs7";
+
 	public OpenShiftNodejsLaunchController() {
 		super();
 	}
 
 	@Override
 	protected void startDebugging(OpenShiftServerBehaviour beh, DebugContext context, IProgressMonitor monitor) {
+		// https://issues.jboss.org/browse/JBIDE-26408
+		Dialogs.INSTANCE.warn("Node.js 8+ unsupported	", 
+				"Debugging is only supported up to Node.js 7. If your Node.js is newer, debugging will not work.\n"
+				+ "If using an OpenShift template to create the application, set 'NODEJS_PROPERTY' to '6'", 
```

Comment:
Why do we recommand Node 6 when the text claims we support up to Node 7 ?

Reply:
I amended this change to your commit.